components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2961,6 +2968,13 @@ where
 
         PEER_ADMIN_CMD_COUNTER.commit_merge.success.inc();
 
+        let range = CacheRange::from_region(&region);
+        info!(
+            "evict range due to commit merge";
+            "range" => ?range,
+        );
+        ctx.engine.evict_range(&range);
```

Comment:
I'm thinking about if both source region and target region already execute evict in the `exec_prepare_merge`, is it necessary here to execute evict_range in the `exec_commit_merge` again?

Reply:
PrepareMerge is a source-region only command, so target regions do not call `evict_range` before CommitMerge.